### PR TITLE
Feature/product model

### PR DESCRIPTION
**Pull request by velenyx**
Сделал базовый каркас модели Product без связей, связи будут дополняться. Нужные связи задокументированы

### CODE REVIEW

**Review by zinuu4 — DISMISSED** [review]

**Review by zinuu4 — COMMENTED** [review]
Looks great, but lacks similar products and products which buy together

**Comment by velenyx** [comment]
> Looks great, but lacks similar products and products which buy together

This is not so important at the current stage, it will be done a little later on

**Review by zinuu4 — COMMENTED** [review]
Looks great, but lacks similar products and products which buy together

**Review by zinuu4 — CHANGES_REQUESTED** [review]
Looks great, but lacks similar products and products which buy together

**Review by zinuu4 — CHANGES_REQUESTED** [review]

File: server/src/models/product.model.js
[the remark this answers is not in the document]
Comment (velenyx) [reply server/src/models/product.model.js:1]: This is not so important at the current stage, it will be done a little later on

**Review by Iki-oops — APPROVED** [review]

**Review by zinuu4 — APPROVED** [review]

**Pull request merged**